doc/source/reference/index.rst

```diff
@@ -41,25 +41,22 @@ to import modules/functions/objects from SciPy.
 Guidelines for importing functions from SciPy
 ---------------------------------------------
 
-The scipy namespace itself only contains functions imported from numpy.  These
-functions still exist for backwards compatibility, but should be imported from
-numpy directly.
-
-Everything in the namespaces of scipy submodules is public.  In general, it is
-recommended to import functions from submodule namespaces.  For example, the
-function ``curve_fit`` (defined in scipy/optimize/_minpack_py.py) should be
+Everything in the namespaces of SciPy submodules is public. In general in
+Python, it is recommended to make use of namespaces. For example, the
+function ``curve_fit`` (defined in ``scipy/optimize/_minpack_py.py``) should be
 imported like this::
 
+  import scipy
+  result = scipy.optimize.curve_fit(...)
+
+Or alternatively one could use the submodule as a namespace like so::
+
   from scipy import optimize
   result = optimize.curve_fit(...)
 
-This form of importing submodules is preferred for all submodules except
-``scipy.io`` (because ``io`` is also the name of a module in the Python
-stdlib)::
-
-  from scipy import interpolate
-  from scipy import integrate
-  import scipy.io as spio
+.. warning:: For ``scipy.io`` prefer the use of  ``import scipy``
```

Comment:
This shouldn't be an admonition I think, certainly not a warning. Can you change to either `.. note::` or a plan sentence?